seatunnel-translation/seatunnel-translation-spark/src/main/java/org/apache/seatunnel/translation/spark/source/batch/ParallelBatchPartitionReader.java

```diff
@@ -92,7 +92,7 @@ protected void prepare() {
             this.internalSource.open();
         } catch (Exception e) {
             running = false;
-            throw new RuntimeException("");
+            throw new RuntimeException(e);
```

Comment:
It's better to add an exception message?

Reply:
Thanks for your opinion, updated.